mantidproject/mantid
# Wiwk2rst

**Description of work.**
This PR adds a few extra features to the `wiki2rst` tool

* Updates the internal hyperlinks - `pandoc` defaults do not link properly
* Allow adding of a handle to the top of the page, so that it can be referenced
* Allow adding of a heading to the page 

**To test:**

Try running the code on some of the Mantid mediawiki pages and compile the output to see if it works.

* `wiki2rst.py --add_heading --add_handle -o python_in_mantid.rst Python_In_Mantid` Should produce an `rst` file with both a heading and a handle at the top.
* Try with different combinations of handle/heading on/off
* Check that internal hyperlinks look correct
* Add the generated page to the Sphinx `ToC` by adding to an `index.rst` file
* Build the `docs-html` target and see if the page builds correctly




*There is no associated issue.*



*This does not require release notes* because **this is developer documentation**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
